### ISSUE TITLE
`importsizeview` returns "Expression tree too large" error when selecting KA > Mathematics topic in import workflow

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#7686]
### Observed behavior

I select the whole "Mathematics" topic and the UI crashes

![image](https://user-images.githubusercontent.com/10248067/98742943-1ae74d00-2364-11eb-89f3-5223cf056527.png)

The server returns a 500, with this error message: 

```
POST
Request URL: http://localhost:8000/device/api/importexportsizeview

OperationalError at /device/api/importexportsizeview
Expression tree is too large (maximum depth 1000)
```

More traceback

```
Traceback:  

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  64.                 return self.cursor.execute(sql, params)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py" in execute
  328.         return Database.Cursor.execute(self, query, params)
  
    
      The above exception (Expression tree is too large (maximum depth 1000)) was the direct cause of the following exception:
    
  

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/views/decorators/csrf.py" in wrapped_view
  58.         return view_func(*args, **kwargs)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/rest_framework/views.py" in dispatch
  495.             response = self.handle_exception(exc)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/rest_framework/views.py" in handle_exception
  455.             self.raise_uncaught_exception(exc)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/rest_framework/views.py" in dispatch
  492.             response = handler(request, *args, **kwargs)

File "/Users/jon/Github/kolibri/kolibri/plugins/device/api.py" in post
  154.         _, total_file_size = calculate_files_to_transfer(nodes_for_transfer, available)

File "/Users/jon/Github/kolibri/kolibri/core/content/utils/import_export_content.py" in calculate_files_to_transfer
  114.         files_to_transfer.aggregate(Sum("file_size"))["file_size__sum"] or 0

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/models/query.py" in aggregate
  351.         return query.get_aggregation(self.db, kwargs.keys())

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/models/sql/query.py" in get_aggregation
  480.         result = compiler.execute_sql(SINGLE)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/models/sql/compiler.py" in execute_sql
  899.             raise original_exception

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/models/sql/compiler.py" in execute_sql
  889.             cursor.execute(sql, params)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  79.             return super(CursorDebugWrapper, self).execute(sql, params)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  64.                 return self.cursor.execute(sql, params)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/utils.py" in __exit__
  94.                 six.reraise(dj_exc_type, dj_exc_value, traceback)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/utils/six.py" in reraise
  685.             raise value.with_traceback(tb)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/utils.py" in execute
  64.                 return self.cursor.execute(sql, params)

File "/Users/jon/.local/share/virtualenvs/kolibri-UH8N_Uf0/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py" in execute
  328.         return Database.Cursor.execute(self, query, params)

Exception Type: OperationalError at /device/api/importexportsizeview
Exception Value: Expression tree is too large (maximum depth 1000)
```

I suspected this might have been caused by #7555, and confirmed that the bug does not present on `0.14.4-weekly3`, but does on `0.14.4-weekly4`

### Expected behavior

Selecting the checkbox on large topics should work without an error.

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

**Comment by rtibbles** [comment]
> Looks like the right fix - this may give us problems with query size for complex queries, but we need to handle that better anyway.

I dislike being correct sometimes.

**Comment by jonboiser** [comment]
I think the fix for this is to simply revert #7555 and figure out a way to fix the thumbnail issue in a way parallel to the core files-to-import query. It seems that expanding the "get files for import" query to include these intermediate topics just makes the query too large.
